provider/gce/google/instance.go

```diff
@@ -175,14 +179,23 @@ func (gi Instance) Metadata() map[string]string {
 	return gi.InstanceSummary.Metadata
 }
 
+// NetworkInterfaces returns the details of the network connection for
+// this instance.
+func (gi Instance) NetworkInterfaces() []*compute.NetworkInterface {
+	return gi.InstanceSummary.NetworkInterfaces
```

Comment:
should this be something that we copy() just in case someone tries to mutate the result value?